UploadDaemon/Configuration/ConfigParser.cs

```diff
@@ -186,4 +186,4 @@ public static YamlConfig Parse(string yaml)
             return result;
         }
     }
-}
+}
```

Comment:
I assume all of these are Linux/Windows line ending changes. What do we actually use in this project? Can we somewhere define it in the project settings which are transferred via git?